docs/responses/source.rst

```diff
@@ -6,100 +6,47 @@ Give us the source of the status.
 .. note::
     The API equivalent is :code:`status_source`.
 
+
 HTTP Code
 ^^^^^^^^^
 
 This source is returned when **all the following cases** are met:
 
-- We can't extract the expiration date from :func:`PyFunceble.lookup.whois.WhoisLookup.request`.
+- We can't extract the expiration date from
+  :func:`PyFunceble.lookup.whois.WhoisLookup.request`.
 - The :code:`INACTIVE` status is the one returned by other methods.
-- :func:`PyFunceble.lookup.http_code.HTTPCode.get` outputs is different from the default one
-  (:code:`XXX`) and the other methods gives the :code:`INACTIVE` status.
+- :func:`PyFunceble.lookup.http_code.HTTPCode.get` outputs is different from
```

Comment:
Not your fault but let's update it!

:class:`PyFunceble.query.http_status_code.HTTPStatusCode` should be correct now 😄 

In other words (previous is weird): `func` is now `class` and `PyFunceble.lookup.http_code.HTTPCode.get` is now `PyFunceble.query.http_status_code.HTTPStatusCode`.

Reply:
Could you throw these into a [todo](https://github.com/spirillen/PyFunceble/issues)?

Reply:
https://github.com/funilrys/PyFunceble/projects/7

Reply:
Should have been fixed

/move to done https://github.com/funilrys/PyFunceble/projects/7#card-52531238